LICENSE-binary

```diff
@@ -300,8 +300,8 @@ MIT License
 
 argparse4j-0.7.0, see: licenses/argparse-MIT
 jopt-simple-5.0.4, see: licenses/jopt-simple-MIT
-slf4j-api-1.7.30, see: licenses/slf4j-MIT
-slf4j-log4j12-1.7.30, see: licenses/slf4j-MIT
+slf4j-api-1.7.36, see: licenses/slf4j-MIT
+slf4j-reload4j-1.7.36, see: licenses/slf4j-MIT
```

Comment:
Since `slf4j-api-1.7.36` uses reload4j anyways, we do not need to use `slf4j-reload4j-1.7.36` anymore. Could you remove it here?

Reply:
It is used and is included in the libs folder

Reply:
Ah, I confirm that it is in the libs folder. I did not expect it! Thank you for checking!

Reply:
No probs. I've double checked the docs (https://www.slf4j.org/manual.html) slf4j-api -> (slf4j-log4j12 ->) slf4j-reload4j -> reload4j